src/Microsoft.SqlTools.Dmp.Contracts/Hosting/ServerCapabilities.cs

```diff
@@ -0,0 +1,67 @@
+//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+namespace Microsoft.SqlTools.Dmp.Contracts.Hosting
+{
+    public class LanguageServiceCapabilities
```

Comment:
Doc comment indicating this contains the Language Service-related settings and a link to their repository / spec would be handy.